kubernetes/charts/weblogic-domain/README.md

```diff
@@ -0,0 +1,115 @@
+# Weblogic Domain Helm Chart
+This chart creates a single Weblogic domain on a [Kubernetes](https://kubernetes.io/) cluster using the 
+[Helm](https://github.com/kubernetes/helm) package manager.
+
+## Prerequisites
+- Kubernetes 1.7.5+ with Beta APIs enabled
```

Comment:
This seems confusing.  Why mention Beta API's?  We have a statement somewhere else with details of supported K8s versions.  I can't remember if we are even saying that we still support 1.7.x.  If so, it would be less confusing to have the list of supported versions and then say something like, "use of K8s 1.7.5 requires enabling Beta APIs".